src/Components/Carousel/Carousel.js

```diff
@@ -84,12 +85,12 @@ const Carousel = () => {
           </button>
         </div>
         <div className="curouselPictures">
+        
           {data &&
             newData.map((e) => {
               return (
-                <div className="wrapper" key={uuidv4()}>
+                <div className="wrapper" key={e.explanation}>
```

Comment:
This would be a good place to do some destructuring. Also, try to define variables with some meaningful name, "e" does not tell us what that variable represents. In most of the cases e is shorten of event, so it's a little bit confusing